app/scripts/controllers/dropboxdatastore.coffee

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+###*
+ # @ngdoc function
+ # @name swarmApp.controller:DropboxdatastoreCtrl
+ # @description
+ # # DropboxdatastoreCtrl
+ # Controller of the swarmApp
+###
+
+#angular.module('swarmApp').controller 'OptionsCtrl', ($scope, $location, options, session, game, env, $log, backfill) ->
+
+angular.module('swarmApp').controller 'DropboxdatastoreCtrl', ($scope, $log , dropboxdatastore, env , dropstoreClient, session) ->
+
+    #$scope.dropstore = dropstoreClient
+    _datastore = null
+    $scope.env = env
+
+
+    $scope.awesomeThings = [
+      'HTML5 Boilerplate'
+      'AngularJS'
+      'Karma'
+    ]
+    $log.debug 'hello world:' ;
+
+    $scope.app_key = 'k614nfbt1fp1mj7';
+    $scope.oauth_token = 'ajamBZAOqM4AAAAAAAABsOkiHt2xjk1hOjloitZmn_dx9s6wS1plhTzICsTBLFHq';
```

Comment:
well normally you dont need oath_toeken.
normally you just call authenticate and you get the token by a post back.. but i think that only works with a public url..  hence the pasting of tokens for the moment.